src/keystats/keystatsContainer.ts

```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux'
+
+import { GlobalState } from '../types';
+import KeyStats from './keystats'
+
+const mapStateToProps = (state: GlobalState) => ({
+  keystats: state.keystats
+})
+
+export type KeyStatsContainerStateProps = ReturnType<typeof mapStateToProps>
```

Comment:
Maybe just `KeyStatsStateProps`?

Reply:
I would say `[ComponentName]` + `StateProps` is a good convention. If you start shortening just because there are too many words then you lose consistency across the project.